### ISSUE TITLE
Error resolving include-after- unable to open file js.html

### ISSUE THREAD

**Issue by ajsmit** [quarto-dev/quarto-cli#5073]
### Bug description

Good day,

My system specifications are:
* MacOS Ventura 13.3
* M1 Mac
* Quarto v1.3.302, to which I upgraded from the previous version, v1.2.475
* RStudio Version 2023.03.0+386

After the upgrade I encounter the folowing error when I render a Quarto document (with R code included) to html from within RStudio and I have been unable to find a solution or even a mention of this problem elsewhere on the internet:

```
ERROR: Error resolving include-after- unable to open file js.html
```

The website being rendered is https://tangledbank.netlify.app/, which was made with Quarto v1.2.475 within RStudio. I have not tried to render the website Quarto v1.3.302 outside of RStudio.

The output of `quarto check` is here:

```
[✓] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.1: OK
      Dart Sass version 1.55.0: OK
[✓] Checking versions of quarto dependencies......OK
[✓] Checking Quarto installation......OK
      Version: 1.3.302
      Path: /Applications/quarto/bin

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....OK
      Version: 3.9.13 (Conda)
      Path: /Users/ajsmit/opt/anaconda3/bin/python
      Jupyter: 4.11.1
      Kernels: python3

[✓] Checking Jupyter engine render....OK

[✓] Checking R installation...........(None)

      Unable to locate an installed version of R.
      Install R from https://cloud.r-project.org/
```

Don't believe the output that says that a version of R cannot be found on my system--it is there and it works.

Herewith a reproducible example:

```
---
date: "2021-01-01"
title: "4. A problem file"
subtitle: "Getting familiar with data handling in R"
bibliography: ../../references.bib
csl: ../../marine-biology.csl
format:
  html: 
    toc-location: right
---

# My heaidng

My text. This problem occurs whether or not I include code chunks.
```

Has this been encountered before? And if so, are you aware or any solutions?

I am unsure whether this an issue on my end, or something the developers should know about.

PS: The Checklist asks for a .qmd file that reproduces the problem, but the system does not actually allow .qmd files to be uploaded.

Thank you,
AJ Smit

### Checklist

- [X] Please include a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.
- [X] Please [format your issue](https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us) so it is easier for us to read the bug report.
- [X] Please document the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [X] Please document the operating system you're running. If on Linux, please provide the specific distribution.
- [X] Please provide the output of `quarto check` so we know which version of quarto and its dependencies you're running.

**Comment by cderv** [comment]
> Don't believe the output that says that a version of R cannot be found on my system--it is there and it works.

Just a side note: Can you open an issue about this ? We should fix this - if R is there, not found by `quarto check` but then it works otherwise, this is definitely an issue we should look into ! 

> Herewith a reproducible example:

Are the  `bibliography` and `csl` important to the issue ? We don't have so the example is not really reproducible. 

> The website being rendered

Does the issue is happening with a document standalone outside of the website project or only inside the website project ? 
I suppose you get the issue inside the website : https://github.com/ajsmit/tangled_bank - I'll try to reproduce with that. 

> PS: The Checklist asks for a .qmd file that reproduces the problem, but the system does not actually allow .qmd files to be uploaded.

Providing the `.qmd` as code in the issue body is fine. Other a github project also, or a zip file should be uploadable.

**Comment by ajsmit** [comment]
Apologies for including the BiBTeX source and style.

I tested a clean project set up specifically to try and isolate the problem. I narrowed it to a line entry in my `_quarto.yml` file. That line entry pointed to the offending `js.html` file. Calling this file did not create an issue in Quarto  v1.2.475, but it was picked up by v1.3.302, and this caused the error.

Specifically, it was included in the `_quarto.yml` file as

```
format:
  html:
    include-after-body: js.html
```

Removing it solved the issue. 

Thank you for looking into this, and apologies for it not being a serious issue on your side (but I suppose that's a good thing).  I will open the issue on `quarto check` that complains about not finding R when it is indeed present.

Kind regards,
AJ
